v-next/hardhat/src/internal/cli/init/template.ts

```diff
@@ -0,0 +1,83 @@
+import path from "node:path";
+
+import {
+  exists,
+  getAllFilesMatching,
+  isDirectory,
+  readdir,
+  readJsonFile,
+} from "@ignored/hardhat-vnext-utils/fs";
+import {
+  findClosestPackageRoot,
+  PackageJsonNotFoundError,
+  type PackageJson,
+} from "@ignored/hardhat-vnext-utils/package";
+
+export interface Template {
+  name: string;
+  packageJson: PackageJson;
+  path: string;
+  files: string[];
```

Comment:
Maybe adding a comment or making it explicit in the name that these paths are relative and the above one is absolute?